recipes-support/syslog-ng/files/syslog-ng.conf.sysvinit

```diff
@@ -1,4 +1,4 @@
-@version: 3.31
+@version: 3.36
```

Comment:
Is it possible to have a placeholder version text here, which gets replaced with the recipe version at build time? Then, we wouldn't have to update this every release.

Reply:
I like that idea. I'll take a stab at it.